content/docs/1.5.0/references/settings.md

```diff
@@ -98,6 +99,12 @@ Define the policy to use when a node with the last healthy replica of a volume i
 
 Longhorn will generate system snapshot during replica rebuild, and if a user doesn't setup a recurring snapshot schedule, all the system generated snapshots would be left in the replica, and user has to delete them manually, this setting allow Longhorn to automatically cleanup system generated snapshot before and after replica rebuild.
 
+#### Automatically Cleanup Recurring Job Backup Snapshot
```

Comment:
Suggested wording: Automatically Clean Up Outdated Snapshots of Recurring Backup Jobs

Reply:
Moved to 1.6.0 and Modified. Thanks for suggestions.